conversations/finaid/staff_panel.py

```diff
@@ -0,0 +1,66 @@
+
+from django.contrib.admin.views.decorators import staff_member_required
+from django.conf.urls import url
+from django.template.response import TemplateResponse
+
+from conference.models import Conference
+from conversations.finaid.api import (
+    get_actionable_finaid_threads,
+    get_all_finaid_threads,
+)
+from conversations.staff_interface import staff_thread, apply_ordering
+from conversations.common_actions import ThreadFilters
+
+# TODO: write a proper decorator here
```

Comment:
What does this mean? It looks like a proper decorator already :)

Reply:
good spot, it means that we need to do more perm checks than just staff member. :)